### PR TITLE
Auth gcloud

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

**Comment by loosebazooka** [comment]
@patflynn @salguarnieri

File: src/main/groovy/com/google/appengine/AppEnginePlugin.groovy [hunk 1]
```diff
@@ -722,18 +722,10 @@ class AppEnginePlugin implements Plugin<Project> {
...
-        else {
-            project.afterEvaluate {
-                if (project.plugins.hasPlugin(EclipsePlugin)) {
-                    EclipseModel model = project.extensions.getByType(EclipseModel)
-                    model.classpath.plusConfigurations += [functionalTestRuntimeConfiguration]
-                }
+        project.afterEvaluate {
+            if (project.plugins.hasPlugin(EclipsePlugin)) {
+                EclipseModel model = project.extensions.getByType(EclipseModel)
+                model.classpath.plusConfigurations += [functionalTestRuntimeConfiguration]
```
Comment (salguarnieri) [inline src/main/groovy/com/google/appengine/AppEnginePlugin.groovy]: Is there any convention about line length?

File: src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy [hunk 2]
```diff
@@ -46,7 +47,7 @@ class GCloudConfigurable {
     }
 
     void args(Map<String, ?> args) {
```
Comment (salguarnieri) [inline src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy]: Why the mix of ? and Object?

Comment (loosebazooka) [inline src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy]: Hrmm, I think it was to enable sending in Map<String, String> or Map<String, Boolean> or whatever as a parameter list. Bad idea?

Comment (loosebazooka) [inline src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy]: But needed the base map to be Map<String, Object> for some other reason.

Comment (loosebazooka) [inline src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy]: Yeah, so when doing map1 + map2 using Map<String, ?> (https://github.com/GoogleCloudPlatform/gradle-appengine-plugin/blob/master/src/main/groovy/com/google/gcloud/GCloudPlugin.groovy#L91) is an unchecked warning, this gets rid of that by ensuring the maps are compatible.

Comment (loosebazooka) [inline src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy]: But maybe, I should have standardized against the wildcard. I'll take another look

Comment (salguarnieri) [inline src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy]: What was your final decision. All of the reasoning makes sense.

Comment (loosebazooka) [inline src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy]: I'm sticking with the current implementation.

**Comment by loosebazooka** [comment]
I think the refresh token task is broken. Need to update

**Comment by loosebazooka** [comment]
no more gcloud in here.

**Pull request closed without merging**